community.md

```diff
@@ -30,6 +30,11 @@ channels:
     src: assets/community/mail-list.svg
     alt: Mailing list
     url: https://groups.google.com/forum/?pli=1#!forum/jupyter-research-facilities
+  - title: JupyteLabr Matrix Chatroom
+    description: Matrix Chatroom for Jupyter.
+    src: assets/community/matrix-logo.svg
+    alt: Matrix logo
+    url: https://matrix.to/#/#jupyterlab_jupyterlab:gitter.im
```

Comment:
What would duplicate gitter, which is no longer used since everyone moved to Zulip and gitter was just dropped from the list in #800